modules/opengl/src/buffer/framebufferobject.cpp

```diff
@@ -78,7 +78,7 @@ void FrameBufferObject::defineDrawBuffers() {
 }
 
 void FrameBufferObject::deactivate() {
-    if (isActive()) {
+    if (isActive() && (static_cast<GLuint>(prevFbo_) != id_)) {
```

Comment:
Should the same thing be done in activate?

Reply:
potentially yes. The [OpenGL specs](https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glBindFramebuffer.xhtml) are not referring to binding the same fbo.

Reply:
addressed in 23d2fbecb6fb40619e99ea94dc8b49243c356f16